README.md

```diff
@@ -30,13 +30,26 @@ or e.g. install a snapshot from a
 use `js-mode` as the major mode. To make use of the JS2 AST and the
 packages that integrate with it, we recommend `js2-minor-mode`. See
 the corresponding [instructions in the
-Commentary](https://github.com/mooz/js2-mode/blob/ae9fea81cf9ed80f403c161fde4d8aff1ceb3769/js2-mode.el#L57).
+Commentary](https://github.com/mooz/js2-mode/blob/6cc0a8a87a9178e44c51203e1fe34c57f246a7de/js2-mode.el#L60).
 
 `js-mode` in Emacs 27 includes full support for syntax highlighting
-and indenting of JSX syntax.
+and indenting of JSX syntax. Try it out!
 
-[rjsx-mode](https://github.com/felipeochoa/rjsx-mode/) is an
-alternative option which comes with certain tradeoffs.
+Until [#527](https://github.com/mooz/js2-mode/issues/527) is
```

Comment:
I don't really mind either way, but... should this go into the wiki instead?

This section is getting a bit long, and it's not like all of our users need to read all this.

Also: if #527 is resolved, we'll probably rewrite the whole recommendation, including the use of the minor mode.

Reply:
Less is more.  I’ve removed my changes here.